examples/rust-examples/mips64/dynamic-memory/README.md

```diff
@@ -0,0 +1,30 @@
+# Rust example for mips64 architecture
+
+Presented here is a straightforward Rust example utilizing Martos with timer usage.
+
+## How to install dependencies
+
+Below is an illustrative example demonstrating the installation of building toolchains on a Linux (Ubuntu/Debian):
+```
+apt update && apt install curl build-essential lld
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y 
+echo 'PATH="${PATH}:/root/.cargo/bin"' >> $GITHUB_PATH 
+. "/root/.cargo/env" 
```

Comment:
For example, you take clear ubuntu. Do you really need this commands? Maybe they are necessary only for ci in github actions...